std_msgs/String[] strings
